src/messages/sq.json

```diff
@@ -1,6 +1,7 @@
 {
 	"common": {
-		"language": "Gjuha"
+		"language": "Gjuha",
+		"produkte": "Produkte"
```

Comment:
These messages should be the exactly same json files with only different values. Think of each file as a dictionary. For example;
`en.json` --> a dictionary from English to English
`sq.json` --> a dictionary from English to Albanian
`sr.json` --> a dictionary from English to Serbian

So keys of the object are always English, the values will be the same word in the respective language.

Reply:
Please re-check all locale json files accordingly